ext/search_kit/ang/crmSearchAdmin/compose.html

```diff
@@ -74,11 +74,11 @@
     </ul>
   </div>
 </div>
-<details id="crm-search-build-functions">
+<details id="crm-search-build-functions" class="crm-search-fieldTransforms">
```

Comment:
This still has camelCase. OTOH it's also going to get renamed from "Field Transformation" to "Select Fields" soon.